plugin-attribution/src/test/java/com/mapbox/maps/plugin/attribution/AttributionDialogManagerImplTest.kt

```diff
@@ -74,7 +81,9 @@ class AttributionDialogManagerImplTest {
     val button =
       attributionDialogManagerImpl.telemetryDialog!!.getButton(DialogInterface.BUTTON_POSITIVE)
     assertNotNull(button)
+    Shadows.shadowOf(Looper.getMainLooper()).pause()
```

Comment:
Perhaps I'm blind but how did this test pass before? I don't see you change actually `AttributionDialogManagerImpl` in this PR, just test for it.

Reply:
Relates to roboeletric bump, read more about it in http://robolectric.org/blog/2019/06/04/paused-looper/